moja-kuhinjica-frontend/src/pages/mealReservation/index.tsx

```diff
@@ -43,10 +52,37 @@ const MealReservation = (): JSX.Element => {
         }
     }, [windowWidth])
 
+    const isCartEmpty = (): boolean => {
+        return cartItems.length === 0
```

Comment:
Remove return and curly brackets

Reply:
i think you can return only `!cartItems?.length` instead `cartItems.length === 0`